projects/extension/src/components/ClientItem.tsx

```diff
@@ -117,23 +106,12 @@ const ClientMenu: React.FunctionComponent<Network | Parachain> = ({isKnown, name
 
         <Typography variant='overline'>Chainspec</Typography>
         <InputWrap>
-
-          <InputText readOnly defaultValue={chainspecPath}/>
           <Divider orientation='vertical' flexItem/>
           <InputButton>
             <InsertDriveFile fontSize='small'/>
           </InputButton>
           <Divider orientation='vertical' flexItem/>
 
-          { isKnown
-          ? <InputButton>
-            <Publish fontSize='small'/>
-          </InputButton>
-          : <InputButton>
-            <SystemUpdateAlt fontSize='small' />
-          </InputButton>
-          }
-
         </InputWrap>
```

Comment:
These changes are clearly unrelated to the ConnectionManager/AppMediator change, aren't they?

Reply:
These ones I am already putting aside on different PR - But there is a slight alternation of UI (on options.tsx) that needs to be in this PR (related to the tabId introduced in AppProps), which is what I was referring to, at my comment above